src/middleware.ts

```diff
@@ -76,6 +76,6 @@ export function createMiddleware(
           next()
         },
       },
-    )
+    ).catch(next)
```

Comment:
Is there a way for us to forward that error in the 500 response message body? Something like:

```
Response.json({
  name: error.name,
  message: error.message,
  stack: error.stack
}, { status: 500 })
```

Reply:
How the error is returned is up to the error middleware. The [default error handler](https://expressjs.com/en/guide/error-handling.html#the-default-error-handler) in Express forwards the error as an HTML response.
We could either provide our own error middleware in `server.ts` or return a 500er response directly in this middleware instead of passing it along the middleware chain.
I think the latter might be to restrictive for projects that only integrate the MSW middleware in their server. It prevents them from formatting the error with their own middleware.

Reply:
Oh, that makes more sense! Thanks for clarifying.